stubs/Pillow/PIL/Image.pyi

```diff
@@ -153,9 +161,11 @@ class Image:
     palette: Any
     info: dict[Any, Any]
     readonly: int
-    pyaccess: Any
+    pyaccess: PyAccess | None
     is_animated: bool  # not present on all Image objects
     n_frames: int  # not present on all Image objects
+    # Can never be None, but can be missing
+    encoderconfig: tuple[Incomplete, ...] | _NotImplementedType
```

Comment:
We usually just add the item with a comment:

```suggestion
    # Only defined after a call to save().
    encoderconfig: tuple[Incomplete, ...]
```

Alternatively, we could just leave it out, as this looks like an internal field.